css/styles.css

```diff
@@ -50,4 +51,25 @@ main .page-container {
 input {
   outline: none;
 }
-/* Search Style */
+
+/* Global Card Style */
+
+.card-style .card {
+  background-color: var(--background-color);
+  border: 1px solid var(--secondary-text-color);
+  margin: 0;
+}
+
+.card-image-container {
+  height: 250px;
```

Comment:
Try to use rem or any other responsive units as much as possible.

Reply:
Done the Changes 
Thank You for Review